common/src/main/java/com/flipkart/flux/FluxRole.java

```diff
@@ -0,0 +1,7 @@
+package com.flipkart.flux;
+
+public enum FluxRole {
```

Comment:
minor: Should we call this FluxRuntimeRole indicating the role is regarding the runtime env?